main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/prometheus/common/log"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var (
+	version = "0.0.1"
```

Comment:
you can let this as `master` or `dev`, and pass it down via ldflags (`go build -ldflags=-X main.version=VERSION`).

if you use goreleaser, it will do it automagically for you actually :)

Reply:
For now i'll leave it as `dev` and add goreleaser in another PR